client/Client.py

```diff
@@ -39,6 +43,42 @@ def join_queue(self):
             self.socket.shutdown(socket.SHUT_RDWR)
             self.socket.close()
 
+    def handle_threads(self):
+        try:
+            while True:
+                end_threads = 1
+                for connection_thread in self.connections_threads:
+                    if connection_thread.status.is_connected == -1:
+                        connection_thread.status.to_kill = 1
+                    if connection_thread.status.is_connected != -1:
+                        end_threads = 0
```

Comment:
Consider using more informative values than -1, 0, 1 (ie. strings describing what to expect)

Reply:
Went for strings in is_connected property, but I'm going to leave to_kill and end_threads as boolean types, changing to True/False though.